index.html

```diff
@@ -1,268 +1,292 @@
-                            <p> <i class="fa fa-chevron-right" aria-hidden="true"></i></p>                            
-                        </div>
-                        <hr id="line-1">
-
-                    </li>
-
-                    <li>
-                        <div class="arrow"> 
-                            <p class="skill">Skills</p> <p> <i class="fa fa-chevron-right" aria-hidden="true"></i> </p>
-                        </div>
-                        <hr id="line-1">
-                    </li>
-                </ul>        
-            </div>
-
-            <div class="form-container">
-                <img class="back-spot" src="Images/back-print.png" alt="" />
-                
-                <div class="form-bottom">
-                    <h2>Contact me</h2>
-                    <p>If you have an application you are interested in developing, a feature that you need built or a project that needs coding. I’d love to help with it!</p>
-
-                    <form action="https://formspree.io/f/xqkoyoly" method="post">
-                        <input type="text" maxlength="30" id="name" class="first" placeholder="Lorem Ipsum" required /><br />
-                        <input type="email" name="email" required id="email" class="second" placeholder="Lorem Ipsum" /> <br />
-
-                        <textarea type="text" maxlength="500" cols="1" rows="10" class="message" required placeholder="Write your message here..." ></textarea><br />
-
-                        <button type="submit" href="#" class="btn">Get in touch </button>
-                    </form>
-
-                    <hr>
-                </div>
-            </div>
-        </section>
-    </section>
-
-</body>
-
-</html>
\ No newline at end of file
+<!DOCTYPE html>
+<html lang="en">
+    <head>
+        <meta charset="UTF-8" />
+        <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+        <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+        <title>Zaf's Portfolio</title>
+        <link rel="stylesheet" href="style.css" />
+        <link
+        rel="stylesheet"
+        href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/6.4.0/css/all.min.css"
+        integrity="sha512-iecdLmaskl7CVkqkXNQ/ZH/XLlvWZOJyj7Yy7tcenmpD1ypASozpmT/E0iPtmFIB46ZmdtAc9eNBvH0H/ZpiBw=="
+        crossorigin="anonymous"
+        referrerpolicy="no-referrer"
+        />
+        <link href="https://fonts.cdnfonts.com/css/poppins" rel="stylesheet" />
+        <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/6.4.0/css/all.min.css" integrity="sha512-iecdLmaskl7CVkqkXNQ/ZH/XLlvWZOJyj7Yy7tcenmpD1ypASozpmT/E0iPtmFIB46ZmdtAc9eNBvH0H/ZpiBw==" crossorigin="anonymous" referrerpolicy="no-referrer" />
+    </head>
+
+    <body>
+
+        <section class="container">
+            <header class="header">
+                <div class="header-left">
+                    <img class="logo" src="Images/my logo.jpg" alt="My logo" />

```

Comment:
- [Optional] Kindly, you mistype the image URL, Please fix it as it has space between `my` and `logo` you have to write it like `Images/My%20Logo.jpg` to display the image, but next time  I will suggest you to use only small case and remove space for static file and directory to follow best practice and protect your self from this type of confusion :+1:

Reply:
@binhussen binhussen
Thanks a lot reviewer. Till now all the reviewers only told me that I have a problem with my images you gave me the solution. I will use your advice from now on. 😃